static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -199,68 +199,81 @@ const buildAppNexusTargeting = once(
         formatAppNexusTargeting(buildAppNexusTargetingObject(pageTargeting))
 );
 
-const buildPageTargeting = once(
-    (): {} => {
-        const page = config.get('page');
-        //
-        const adConsentState: boolean | null = getAdConsentState(
-            thirdPartyTrackingAdConsent
-        );
-
-        // personalised ads targeting
-        const paTargeting: {} =
-            adConsentState !== null ? { pa: adConsentState ? 't' : 'f' } : {};
-        const adFreeTargeting: {} = commercialFeatures.adFree
-            ? { af: 't' }
-            : {};
-        const pageTargets: PageTargeting = Object.assign(
-            {
-                sens: page.isSensitive ? 't' : 'f',
-                x: getKruxSegments(),
-                pv: config.get('ophan.pageViewId'),
-                bp: findBreakpoint(),
-                at: getCookie('adtest') || undefined,
-                si: isUserLoggedIn() ? 't' : 'f',
-                gdncrm: getUserSegments(),
-                ab: abParam(),
-                ref: getReferrer(),
-                ms: formatTarget(page.source),
-                fr: getVisitedValue(),
-                // round video duration up to nearest 30 multiple
-                vl: page.videoDuration
-                    ? (Math.ceil(page.videoDuration / 30.0) * 30).toString()
-                    : undefined,
-                cc: geolocationGetSync(),
-                s: page.section, // for reference in a macro, so cannot be extracted from ad unit
-                pr: 'dotcom-platform', // rendering platform
-                inskin: inskinTargetting(),
-            },
-            page.sharedAdTargeting,
-            paTargeting,
-            adFreeTargeting,
-            getWhitelistedQueryParams()
-        );
-
-        // filter out empty values
-        const pageTargeting: {} = pickBy(pageTargets, target => {
-            if (Array.isArray(target)) {
-                return target.length > 0;
-            }
-            return target;
-        });
+const buildPageTargetting = (adConsentState: boolean | null): Object => {
```

Comment:
`Object` is an alias to `any` and will be deprecated and removed in a future version of Flow. 

`any`, `Object` and `Function` are considered unsafe types and should be avoided as much as possible. 

Do we need to switch to `Object` here? This opts-out of the type checking entirely. Flow also has `mixed` for when you decide to refine the types elsewhere, which I would prefer over `Object or `any`.

I know that Flow types can be a headache, particularly with large objects, but would this work: 

```
const buildPageTargetting = (adConsentState: boolean | null): { [key: string]: mixed } => {
```

Background on change from `Object` to `{}` can be found in https://github.com/guardian/frontend/pull/20453